packages/unit-testing/README.md

```diff
@@ -60,18 +59,18 @@ However, if you want to configure SuiteCloud Unit Testing manually, do the follo
     });
     ```
 8. From the root of your SuiteCloud project folder, run `npm test`
-to run your test. You should see an output similar to following:
+to run your test. You should see an output similar to the following:
     ```
     PASS  __tests__/sample-test.js
     Basic jest test with simple assert
         √ should assert stings are equal (2ms)
     ```
->💡 If you want to run tests with coverage, from the root of your SuiteCloud project folder, run `npm test --coverage`.
+>💡 If you want to run tests with coverage, run `npm test --coverage` from the root of your SuiteCloud project folder.
```

Comment:
I'm not sure that we actually provide code coverage. I'll check it and let you know.

Reply:
Confirmed, if we run `npm test --coverage` we have the same output as `npm test` , so there is no test coverage setup.

Reply:
Changed.